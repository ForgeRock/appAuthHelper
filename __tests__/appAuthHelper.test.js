const appAuthHelper = require("../appAuthHelper");

test("appAuthHelper has the expected shape", () => {
    expect(appAuthHelper).toMatchSnapshot();
});