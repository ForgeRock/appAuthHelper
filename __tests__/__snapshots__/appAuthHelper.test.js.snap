// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`appAuthHelper has the expected shape 1`] = `
Object {
  "getTokens": [Function],
  "handleIdentityProxyMessage": [Function],
  "init": [Function],
  "logout": [Function],
  "registerIdentityProxy": [Function],
  "registerXHRProxy": [Function],
  "renewTokens": [Function],
  "whenRenewTokenFrameAvailable": [Function],
}
`;
